content/tutorials/history.textile

```diff
@@ -98,6 +116,16 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/c99d105
 
+blang[php].
+  ```[php]
+    $channel = $ably->channels->get("persisted:sounds");
+    $channel->publish("play", "bark");
+    $channel->publish("play", "meow");
+    $channel->publish("play", "cluck");
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/f2906e4
```

Comment:
I appreciate the other languages have not provided instructions on how to run the example, but I think we should add that please.  See https://github.com/ably/docs/pull/154/files#diff-a551a3cd350dc478774a772f537a027fR127 for an example of how we have done that before.  We'll add instructions for running the examples for Ruby, Node and the browser shortly as well.